app/components/work_package_relations_tab/closest_relation.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+module WorkPackageRelationsTab
+  class ClosestRelation
+    include Comparable
+
+    attr :relation
+
+    delegate :predecessor, to: :relation
+    delegate :created_at, to: :predecessor
```

Comment:
`delegate` supports multiple arguments:
```suggestion
    delegate :predecessor, :created_at, to: :relation
```

Reply:
@cbliard as I've implemented it, we're falling back on `WorkPackage#created_at` and not `Relation#created_at` for cases where soonest starts are the same (or `nil`). 

As such, this does need to be `delegate :predecessor, to: :relation` (I've removed the second `delegate` line as inlining it in `#<=>` makes the intent clearer IMO).

Could you just double-check this correct?

Reply:
Sorry, I'm wrong. You're completly right.